sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Metrics/Metrics.cs

```diff
@@ -25,12 +25,15 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
+using SonarAnalyzer.Common;
 using SonarAnalyzer.Helpers;
 
-namespace SonarAnalyzer.Common.CSharp
+namespace SonarAnalyzer.Metrics.CSharp

```

Comment:
Would probably deserve to be in a separate commit.

Reply:
Not worth spliting now, but I will keep it in mind for future.